charts/agent/Chart.yaml

```diff
@@ -5,7 +5,7 @@ description: Sysdig Monitor and Secure agent
 type: application
 
 # currently matching sysdig 1.12.56
-version: 1.0.2
+version: 1.0.3
```

Comment:
to 1.1.1 maybe?

Reply:
thanks for pointing this out! I'll use 1.3.0 in line with the earlier versioning discussion